xml/compute.xml

```diff
@@ -1607,35 +1631,37 @@
   <title>Profiling and benchmarking libraries and tools</title>
   <para>
    Performance optimization plays an important role in HPC. That applications
-   are run across multiple cluster nodes poses an additional challange.
+   are run across multiple cluster nodes poses an additional challenge.
    &shpca; provides a number of tools for profiling MPI applications and
    benchmarking MPI performance.
   </para>
   <sect2 xml:id="sec2-tool-imb">
    <!-- href="https://fate.novell.com/324155" -->
-   <title>IMB &mdash; Intel&thrdmrk; MPI benchmarks</title>
+   <title>IMB &mdash; Intel* MPI benchmarks</title>
    <para>
-    Intel(R) MPI Benchmarks provides a set of elementary benchmarks that conform
-    to MPI-1, MPI-2, and MPI-3 standard. One can run all of the supported
-    benchmarks, or a subset specified in the command line using one executable
+    Intel* MPI Benchmarks provides a set of elementary benchmarks that conform
```

Comment:
Hm , I wonder if it should be "provide" after all..? (It's a single RPM comprising all benchmarks. But the package contains a set of benchmarks nonetheless.)

```suggestion
    Intel* MPI Benchmarks provide a set of elementary benchmarks that conform
```

Reply:
If we want to use the singular, then I think it needs to be "the Intel MPI Benchmarks package provides..." I am happy to do this, but it is wordier.

Reply:
Good suggestion -- makes sense to me!